taccsite_cms/static/site_cms/js/site.js

```diff
@@ -1,3 +0,0 @@
-'use strict';
-
-console.info('Core `site.js` has no logic.');
```

Comment:
**Original Intent:** Note to self that there is a `site.js` for site-wide JavaScript. _But no site-wide JavaScript has since been needed._